files/zh-cn/web/javascript/guide/regular_expressions/assertions/index.md

```diff
@@ -271,9 +270,9 @@ console.log('This is a First peach in a month.'.match(regex)); // null
 console.log(/\d+(?!\.)/g.exec('3.141')); // [ '141', index: 2, input: '3.141' ]
 ```
 
-### 不同含义的'?!'：断言和范围的组合用法
+### 不同含义的'?!'：断言和字符集的组合用法
 
-不同含义的 `?!` 结合使用[断言](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Assertions) `/x(?!y)/` 和[范围](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Groups_and_Ranges) `[^?!]`。
+在像 `/x(?!y)/` 这样的断言和 `[^?!]` 这样的[字符类](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Character_classes)中，`?!` 组合具有不同的含义。
 
 ```plain
```

Comment:
Maybe we also need to format the code block (please also check the next section and the above sections)

````suggestion
```js
````